pkg/util/converter/prom_test.go

```diff
@@ -55,7 +55,7 @@ func TestReadLimited(t *testing.T) {
 				//nolint:gosec
 				f, err := os.Open(p)
 				require.NoError(t, err)
-				mbr := httpclient.MaxBytesReader(f, i)
+				mbr := http.MaxBytesReader(f, i)
```

Comment:
Nope this is a different thing, I need to be able to import the httpclient package (or this func/pkg to backend-go-sdk/httpclient - if it makes sense to move it there?).

Reply:
or MaxByteReader to grafana-plugin-sdk-go?